### PR TITLE
Unify markdown formatting in network guide

### PR DESCRIPTION
**Pull request by phoracek**
Use syntax highlighting and mark headings consistently with #.

Signed-off-by: Petr Horacek <phoracek@redhat.com>

### CODE REVIEW

File: creation/interfaces-and-networks.md [hunk 1]
```diff
@@ -503,43 +524,43 @@ pod's IPv6 address.
 Unlike in IPv4, the configuration of the IP address, default route, and DNS is
 not automatic; it should be configured via cloud init, as shown below:
 
-<pre>
```
[the remark this answers is not in the document]
Comment (fabiand) [reply creation/interfaces-and-networks.md:506]: Anything to update here?

**Comment by fabiand** [comment]
@phoracek please rebase and we can get this one merged

**Comment by codificat** [comment]
@mazzystr these changes here seem worth taking into account for your current markdown engine work

File: creation/interfaces-and-networks.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (phoracek) [reply creation/interfaces-and-networks.md]: We don't need _italics_ in this snippet anymore. I changed it into the same syntax as all the other blocks

**Comment by phoracek** [comment]
Rebased

**Comment by mazzystr** [comment]
@codificat I'm happy with this pull req.  Rebasing #372 will be easy

Netlify needs to be updated with with new build cmd in relation to admonition
Prow / user-guide-presubmit-build might need an adjustment in relation to admonition

Gotta love interrupt driven work....  I'll handle these bits right now.

**Comment by codificat** [comment]
Ok. As CI is not working at the moment for this repo I'm merging it manually.

**Comment by mazzystr** [comment]
I'm good with that.  Thank you.

**Pull request merged**